app/AnnotationSession.php

```diff
@@ -94,10 +94,11 @@ public function getVolumeFileAnnotations(VolumeFile $file, User $user)
                 // wrap this in a where because the default query already has a where
                 $query->where(function ($query) use ($user) {
                     $query->where(function ($query) {
-                        $query->where('created_at', '>=', $this->starts_at)
+                        $query
+                            ->where('created_at', '>=', $this->starts_at)
                             ->where('created_at', '<', $this->ends_at);
                     })
-                    ->orWhere('user_id', '!=', $user->id);
+                        ->orWhere('user_id', '!=', $user->id);
```

Comment:
And this line should stay at the same level of indentation than the line above.

In total this would be:
```php
// take only labels of this session or any labels of other users
$query->with(['labels' => function ($query) use ($user) {
    // wrap this in a where because the default query already has a where
    $query->where(function ($query) use ($user) {
        $query
            ->where(function ($query) {
                $query->where('created_at', '>=', $this->starts_at)
                    ->where('created_at', '<', $this->ends_at);
            })
            ->orWhere('user_id', '!=', $user->id);
    });
}]);
```

Reply:
Don't you mean something like this?

`$query->with(['labels' => function ($query) use ($user) {
                // wrap this in a where because the default query already has a where
                $query->where(function ($query) use ($user) {
                    $query->where(function ($query) {
                        $query
                            ->where('created_at', '>=', $this->starts_at)
                            ->where('created_at', '<', $this->ends_at);
                    })
                        ->orWhere('user_id', '!=', $user->id);
                });
            }]);`

Or do I miss something out?

Reply:
Don't you mean something like this?

```php
$query->with(['labels' => function ($query) use ($user) {
                // wrap this in a where because the default query already has a where
                $query->where(function ($query) use ($user) {
                    $query->where(function ($query) {
                        $query
                            ->where('created_at', '>=', $this->starts_at)
                            ->where('created_at', '<', $this->ends_at);
                    })
                        ->orWhere('user_id', '!=', $user->id);
                });
            }]);
```

Or do I miss something out?

Reply:
Ignore my latest comments on this, I was a little confused.

I still don't understand why only the third `$query` is changed. What pattern should I look for exactly?

Reply:
The third `$query` is changed because it starts a method chain and opens a multi-line closure. It should be the only instance that the CS Fixer would want to change in its original state. While the first and second `$query` open closures, they do not start method chains. The fourth `$query` starts a method chain but does not open a closure or array.

I think if you only look at it line by line, there is no pattern that can be used to determine which lines should be changed. I'm beginning to think that maybe we have to update these instances manually or we have to disable the indentation checks of CS fixer entirely.

Reply:
I will then make the changes by hand.

Reply:
It's probably easiest to run the CS fixer and then look for the relevant places in the code with `git diff`.